### PR TITLE
[Turnstile] Q4 updates, feature table, ephemeral IDs

### PR DESCRIPTION
**Pull request by patriciasantaana**
### Summary

Documentation on widget limits, offlabel, hostname validation, ephemeral ids, etc.

### Documentation checklist

- [ ] The [documentation style guide](https://developers.cloudflare.com/style-guide/) has been adhered to.

### CODE REVIEW

File: src/content/docs/turnstile/concepts/ephemeral-id.mdx [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+sidebar:
+  order: 4
+
+---
+
+Ephemeral IDs generate a unique short-lived ID that can link behavior to a specific client instead of an IP address without relying on setting any cookies or using similar client-side storage.
+
+When the same visitor interacts with Turnstile widgets from different Cloudflare customers, they receive different Ephemeral IDs for each contact. In attacks where fraudsters attempt to disguise themselves using different IP addresses, Ephemeral IDs detect abuse patterns more accurately than determining whether the visitor is a human or a bot. 
+
+Ephemeral IDs are not unique and have a lifespan of XX seconds/minutes. These IDs cannot be used to track a single visitor over multiple days as they change frequently.
```
Comment (Le0Developer) [inline src/content/docs/turnstile/concepts/ephemeral-id.mdx]:
```suggestion
Ephemeral IDs are not unique and have a short lifespan (think seconds/minutes, not hours). These IDs cannot be used to track a single visitor over multiple days as they change frequently.
```
„XX“ sounds bad. I don’t think „think seconds/minutes“ is perfect, but better

File: src/content/docs/turnstile/concepts/hostname-management.mdx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+title: Hostname management
+pcx_content_type: reference
+sidebar:
+  order: 2
+
+---
+
+By default, all widgets can have up to 10 hostnames associated with a widget. A widget requires at least one hostname to be entered.
+
+Only Enterprise Bot Management and Enterprise Turnstile customers can have this limit increased. Contact your account team to increase your hostname limit.
```
Comment (Le0Developer) [inline src/content/docs/turnstile/concepts/hostname-management.mdx:11]:
```suggestion
Customers with Enterprise Bot Management or Enterprise Turnstile can have this limit increased. Contact your account team to increase your hostname limit.
```
Improved wording imo

a few more thoughts:
- Turnstile is separate from BM now?
- maybe link to sales?
- Also add to FAQ?

File: src/content/docs/turnstile/concepts/hostname-management.mdx [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+- `example.com`
+- `dash.example.com`
+- `cloudflare.com`
+
+When the widget is embedded on a hostname not listed, it will show an error message.
+
+:::note
+
+Enterprise Bot Management customers can remove the hostname validation requirement for a widget. For more information on this feature, contact your account team.
```
Comment (Le0Developer) [inline src/content/docs/turnstile/concepts/hostname-management.mdx]: What about Enterprise Turnstile here? Also this is duplicated by the section right below, so either remove this aside or the next section

File: src/content/docs/turnstile/concepts/widget.mdx [hunk 4]
```diff
@@ -20,49 +20,65 @@ Widgets can be implemented in normal, flexible, or compact sizes.
 
 Refer to [widget sizes](/turnstile/get-started/client-side-rendering/#widget-size) for an example on how to configure flexible or compact mode.
 
-## Managed (recommended)
+:::note[Offlabel]
+Enterprise customers have the ability to remove the Cloudflare branding and customize the Turnstile widget.
```
Comment (Le0Developer) [inline src/content/docs/turnstile/concepts/widget.mdx]: Which Enterprise customers? All or just BM/Turnstile?

File: src/content/docs/turnstile/concepts/widget.mdx [hunk 5]
```diff
@@ -20,49 +20,65 @@ Widgets can be implemented in normal, flexible, or compact sizes.
...
+:::
+
+## Availability 
+
+Free users are limited to 10 widgets per account. 
+
+[Enterprise Bot Management](/bots/get-started/bm-subscription/) customers can increase this limit to any higher number through the widget limit entitlement.
```
Comment (Le0Developer) [inline src/content/docs/turnstile/concepts/widget.mdx]: Missing Enterprise Turnstile mention? And there’s a redundant aside mentioning this exact same thing right below

File: src/content/plans/index.json [hunk 6]
```diff
@@ -2010,6 +2010,58 @@
...
+					"title": "Hostname validation",
+					"link": "/turnstile/concepts/hostname-management/",
+					"free": "10 hostnames per widget",
+					"ent": "200 hostnames per widget. <br> The `any hostname` entitlement removes hostname as a requirement and allows you to use the widget anywhere."
+				},
+				"graphql_analytics": {
+					"title": "Analytics time window",
+					"link": "/turnstile/turnstile-analytics/",
+					"free": "30 days maximum",
+					"ent": "30 days maximum"
```
Comment (Le0Developer) [inline src/content/plans/index.json:2051]: Isn’t the normal enterprise graphql limit 90 days?

File: src/content/plans/index.json [hunk 7]
```diff
@@ -2010,6 +2010,58 @@
...
+			"title": "Turnstile",
+			"link": "/turnstile/",
+			"properties": {
+				"widgets": {
+					"title": "Number of widgets",
+					"link": "/turnstile/concepts/widget/",
+					"free": "10 widgets",
+					"ent": "Unlimited"
+				},
+				"logo": {
```
Comment (Le0Developer) [inline src/content/plans/index.json]: What’s the difference between between logo and offlabel?

**Review by Le0Developer — COMMENTED** [review]
Couple thoughts

File: src/content/docs/turnstile/concepts/ephemeral-id.mdx (hunk 1 above)
Comment (patriciasantaana) [reply src/content/docs/turnstile/concepts/ephemeral-id.mdx]: Thanks for the comments @Le0Developer! The `XX` value is just a placeholder as I still need to verify the actual timeframe.

File: src/content/docs/turnstile/get-started/server-side-validation.mdx [hunk 8]
```diff
@@ -269,6 +270,7 @@ It always contains a `success` property, either true or false, indicating whethe
 - `action` is the customer widget identifier passed to the widget on the client side. This is used to differentiate widgets using the same sitekey in analytics. Its integrity is protected by modifications from an attacker. It is recommended to validate that the action matches an expected value.
 - `cdata` is the customer data passed to the widget on the client side. This can be used by the customer to convey state. It is integrity protected by modifications from an attacker.
 - `error-codes` is a list of errors that occurred.
+- `ephemeral_id` returns the [Ephemeral ID](/turnstile/concepts/ephemeral-id/) in siteverify.
```
Comment (rupert648) [inline src/content/docs/turnstile/get-started/server-side-validation.mdx]: might be worth noting that ephemeral ids are only available for ENT here as well

File: src/content/docs/turnstile/get-started/server-side-validation.mdx [hunk 9]
```diff
@@ -258,7 +258,8 @@ It always contains a `success` property, either true or false, indicating whethe
 	"hostname": "example.com",
 	"error-codes": [],
 	"action": "login",
-	"cdata": "sessionid-123456789"
+	"cdata": "sessionid-123456789",
+	"ephemeral_id": false
```
Comment (Le0Developer) [inline src/content/docs/turnstile/get-started/server-side-validation.mdx]: This seems like an odd example value. It should be `undefined` aka absent if you don’t have access to ephemeral ids and I‘d assume it’s a string/number if you do have access. When would this ever be false?

File: src/content/docs/turnstile/concepts/ephemeral-id.mdx [hunk 10]
```diff
@@ -0,0 +1,19 @@
...
+
+When the same visitor interacts with Turnstile widgets from different Cloudflare customers, they receive different Ephemeral IDs for each contact. In attacks where fraudsters attempt to disguise themselves using different IP addresses, Ephemeral IDs detect abuse patterns more accurately than determining whether the visitor is a human or a bot. 
+
+Ephemeral IDs are not unique and have a lifespan of up to a few days. These IDs cannot be used to track a single visitor over multiple days as they change frequently.
+
+Refer to the [blog post](https://blog.cloudflare.com/turnstile-ephemeral-ids-for-fraud-detection/) for more information.
+
+## Availability
+
+Ephemeral IDs are available to customers with Enterprise Bot Management and Enterprise Turnstile.
```
Comment (rupert648) [inline src/content/docs/turnstile/concepts/ephemeral-id.mdx]:
```suggestion
Ephemeral IDs are available to customers with Enterprise Bot Management and Enterprise Turnstile. Contact your account team to gain access to Ephemeral IDs.
```

**Review by Le0Developer — COMMENTED** [review]
Also as a general question: is an ephemeral id a string or an integer? I’m assuming it’s a string, but I think it’d make sense to explicitly document this somewhere, even if just in the example below.

**Review by Maddy-Cloudflare — APPROVED** [review]

**Pull request merged**